docs/reference/symbolic.rst

```diff
@@ -0,0 +1,71 @@
+
+.. _symbolic_math:
+
+=============
+Symbolic Math
+=============
+
+You can construct a model, for example a CQM, from symbolic math, which is especially
```

Comment:
do we want to write out constrained quadratic model rather than use the abbreviation here? Or alternatively maybe just say more generally `You can construct problems for quantum and hybrid solvers with symbolic math...`

Reply:
Changed to "a constrained quadratic model (CQM)," (rather than bring in solver here for now)